### DISCUSSION TITLE
Mozilla common voice dataset formatter

### DISCUSSION THREAD

**Discussion by Skinzart** [coqui-ai/TTS#3389] [General Q&A]
Hi, I want to try to train a model using a dataset from Mozilla Common Voice, the audio files are in MP3 format,
but if I look at the formatter file for CommonVoice all the paths are replaced from MP3 to WAV.
does this mean I have to convert the files to wav manually? while this contains thousands of audio files
is there any other way?

**Comment by eginhard** [comment]
Yeah, I noticed the same. There this script in the old STT repo to convert the corpus to wav files, but I don't know why this was done: https://github.com/coqui-ai/STT/blob/main/bin/import_cv2.py (see prior discussions https://github.com/coqui-ai/TTS/discussions/1143, https://github.com/coqui-ai/TTS/discussions/121)

The easiest way might be to change the formatter to load mp3.

**Reply by Skinzart** [reply]
Yep, I tried to change the formatter to keep using mp3 but the VITS training process stopped and was hit by an assertion,
so the only way i converted all the audio files to wav.
